ipynb/scripts/remove-blooms.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+import qiime2 as q2
+from biom import load_table, Table
+
+bt = q2.Artifact.load('deblur-fmt-and-icu/table.qza').view(Table)
+
+blooms = []
+with open('newbloom.all.fna', 'r') as f:
```

Comment:
This file should be downloaded via the notebook, I'll note this in the comments.